modules/virt-checking-storage-configuration.adoc

```diff
@@ -0,0 +1,165 @@
+  - apiGroups: [ "cdi.kubevirt.io" ]
+    resources: [ "datavolumes" ]
+    verbs: [ "create", "delete" ]
+  - apiGroups: [ "" ]
+    resources: [ "persistentvolumeclaims" ]
+    verbs: [ "delete" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: storage-checkup-role
+subjects:
+  - kind: ServiceAccount
+    name: storage-checkup-sa
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: storage-checkup-role
+----
+====
+
+. Apply the `ServiceAccount`, `Role`, and `RoleBinding` manifest:
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <storage_sa_roles_rolebinding>.yaml
+----
+
+. Create a `ConfigMap` and `Job` manifest. The config map contains the input parameters for the checkup job.
++
+.Example input config map
+[source,yaml,subs="attributes+"]
+----
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: storage-checkup-config
+  namespace: $CHECKUP_NAMESPACE
+data:
+  spec.timeout: 10m
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: storage-checkup
+  namespace: $CHECKUP_NAMESPACE
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      serviceAccount: storage-checkup-sa
+      restartPolicy: Never
+      containers:
+        - name: storage-checkup
+          image: quay.io/kiagnose/kubevirt-storage-checkup:main
+          imagePullPolicy: Always
+          env:
+            - name: CONFIGMAP_NAMESPACE
+              value: $CHECKUP_NAMESPACE
+            - name: CONFIGMAP_NAME
+              value: storage-checkup-config
+----
+
+. Apply the `ConfigMap` and `Job` manifest in the target namespace to run the checkup:
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <storage_configmap_job>.yaml
+----
+
+. Wait for the job to complete:
++
+[source,terminal]
+----
+$ oc wait job storage-checkup -n <target_namespace> --for condition=complete --timeout 10m
+----
+
+. Review the results of the checkup by running the following command:
++
+[source,terminal]
+----
+$ oc get configmap storage-checkup-config -n <target_namespace> -o yaml
+----
++
+.Example output config map (success)
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: storage-checkup-config
+  labels:
+    kiagnose/checkup-type: kubevirt-storage
+data:
+  spec.timeout: 10m
+  status.succeeded: "true" <1>
+  status.failureReason: "" <2>
+  status.startTimestamp: "2023-07-31T13:14:38Z" <3>
+  status.completionTimestamp: "2023-07-31T13:19:41Z" <4>
+  status.result.defaultStorageClass:
+  status.result.storageProfilesWithEmptyClaimPropertySets:
+  status.result.storageProfilesWithSpecClaimPropertySets:
+  status.result.storageWithRWX:
+  status.result.storageMissingVolumeSnapshotClass: 
+  status.result.goldenImagesNotUpToDate: 
+  status.result.vmsWithNonVirtRbdStorageClass: 
+  status.result.vmsWithUnsetEfsStorageClass: 
+  status.result.vmBootFromGoldenImage: 
+  status.result.vmVolumeClone: 
+  status.result.vmLiveMigration: 
+  status.result.vmHotplugVolume: 
```

Comment:
Please document the `status.results` fields similarly to the other checkups. It can be copied from [upstream doc](
https://github.com/kiagnose/kubevirt-storage-checkup?tab=readme-ov-file#checkup-results-retrieval).